Makefile

```diff
@@ -0,0 +1,13 @@
+OUT := build
+MPYCROSS := /usr/bin/mpy-cross
+MPYC_FLAGS := -v -v -mcache-lookup-bc
+PYS := $(shell find ./ev3dev2 -type f \( -iname "*.py" ! -iname "setup.py" \))
+MPYS := $(PYS:./%.py=${OUT}/%.mpy)
+vpath %.py .
+
+${OUT}/%.mpy: %.py
+	@mkdir -p $(dir $@)
+	${MPYCROSS} ${MPYC_FLAGS} -o $@ $<
+
+install: ${MPYS}
```

Comment:
Maybe call this `micropython-install` so  `install` could do `python3 setup.py install`?

Reply:
Good. Done. But committed in the Console() PR #649.